biomage/rds/migrate.py

```diff
@@ -0,0 +1,95 @@
+import os
+import subprocess
+from pathlib import Path
+
+import click
+
+from ..rds.tunnel import close_tunnel as close_tunnel_cmd
+from ..rds.tunnel import open_tunnel as open_tunnel_cmd
+from ..utils.constants import DEFAULT_AWS_ACCOUNT_ID, DEVELOPMENT, STAGING
+
+IAC_PATH = os.getenv("BIOMAGE_IAC_PATH")
+
+def _migrate(iac_path, migration_env):
+    proc = subprocess.Popen(
+    [
+        "node_modules/.bin/knex",
+        "migrate:latest",
+        "--cwd",
+        iac_path
+    ],
+        cwd=iac_path,
+        env=migration_env
+    )
```

Comment:
Does it show the output from running this?

Reply:
yes, it shows the usual output just like if we run it manually